src/map.cpp

```diff
@@ -4579,12 +4579,12 @@ void map::make_active( item_location &loc )
 void map::apply_in_fridge( item &it, int temp )
 {
     unsigned int diff_freeze = abs(temp - FREEZING_TEMPERATURE);
-    diff_freeze = diff_freeze < 1 ? 1 : diff_freeze;
-    diff_freeze = diff_freeze > 10 ? 10 : diff_freeze;
+    diff_freeze = std::max( static_cast<unsigned int>(1), diff_freeze );
+    diff_freeze = std::min( static_cast<unsigned int>(5), diff_freeze );
 
     unsigned int diff_cold = abs(temp - FRIDGE_TEMPERATURE);
-    diff_cold = diff_cold < 1 ? 1 : diff_cold;
-    diff_cold = diff_cold > 10 ? 10 : diff_cold;
+    diff_freeze = std::max( static_cast<unsigned int>(1), diff_cold );
+    diff_freeze = std::min( static_cast<unsigned int>(5), diff_cold );
```

Comment:
Sorry for bothering. These two lines seems wierd as the above one keeps `diff_cold` and those two change to `diff_freeze`. I'm writing this just so that it can be double-checked.

Reply:
You are right and fix is included in PR #24971 waiting to be merged.